tools/generator/src/Generator+Main.swift

```diff
@@ -39,21 +40,24 @@ extension Generator {
         let workspaceOutputPath: Path
         let projectRootDirectory: Path
         let buildMode: BuildMode
+        let automaticSigningTeamId: String
     }
 
     static func parseArguments(_ arguments: [String]) throws -> Arguments {
-        guard CommandLine.arguments.count == 7 else {
+        guard CommandLine.arguments.count == 8 else {
             throw UsageError(message: """
 Usage: \(CommandLine.arguments[0]) <path/to/project.json> \
 <path/to/xccurrentversions.json> <path/to/bazel/integration/dir> \
 <path/to/output/project.xcodeproj> <workspace/relative/output/path> \
-(xcode|bazel)
+(xcode|bazel) (teamId)
 """)
         }
 
         let workspaceOutput = CommandLine.arguments[5]
         let workspaceOutputComponents = workspaceOutput.split(separator: "/")
 
+        let teamId = CommandLine.arguments[7]
```

Comment:
Since you don't do any processing on this, this can be used directly in the constructor below.